ee/fleetd-chrome/src/background.ts

```diff
@@ -70,7 +75,7 @@ const enroll = async () => {
   };
 
   const { enroll_secret } = await chrome.storage.managed.get({
-    enroll_secret: "Y3nHXcZUBcFJc/7e6V6k1z7RG22rrAnQ",
```

Comment:
Is this a default value to use if the setting is not set via management?

Reply:
Yup